docs/historic-one-time-operations/release56.0_day_plan.md

```diff
@@ -1,58 +0,0 @@
-This is the template we used to coordinate on the 56.0 release day with relman.   It could be updated for the next release.  Please keep in mind that 
```

Comment:
Do we want to clean up stuff in `historic-one-time-operations`? I don't haves strong feelings either way. If so, maybe the other 56 related document there could be removed too.

Reply:
Sure, let's keep them, since afterall, they are located within the "historic" folder. We never know when we'll need to related back to them.